platform/nodejs/tests/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "mocha": true
```

Comment:
Without this we get errors for `beforeEach()`, `describe()`, etc., since these are globals in the Mocha context.